kedro-org/kedro-viz
# Ensure `Created by` is set in Experiment Tracking

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Fixes #911.

## Development notes

It was a one-line fix. The `author` field wasn't being set, and now it is.

## QA notes

Take a look at the Gitpod link and view the most-recent run. The field should be filled in.

## Checklist
